package.json

```diff
@@ -33,21 +33,31 @@
   "homepage": "https://github.com/borisyankov/react-sparklines#readme",
   "devDependencies": {
     "babel-core": "^5.6.15",
+    "babel-jest": "*",
     "babel-loader": "^5.2.2",
     "jest-cli": "^0.4.15",
     "jsx-loader": "^0.13.2",
-    "babel-jest": "*",
     "node-libs-browser": "^0.5.2",
     "react-tools": "*",
     "webpack": "^1.10.0",
     "webpack-dev-server": "^1.10.1"
   },
-  "dependencies": {
+  "peerDependencies": {
+    "react": "^0.13.3"
```

Comment:
Only if you have it in `dependencies`. I've put into `peerDependencies` so it will not be installed twice.